backend/src/rtp.rs

```diff
@@ -1913,9 +1986,10 @@ impl Endpoint {
             *next_outgoing_srtcp_index,
             key,
             salt,
-        )?;
+        )
+        .ok_or(EndpointError::InvalidOutgoingSrtcpIndex)?;
```

Comment:
I don't think you need `Index` in the name. It is the Srtcp itself that is invalid, not the index.

Reply:
Alright that I will remove that !